package.json

```diff
@@ -102,6 +102,7 @@
     "@ionic-native/core": "4.14.0",
     "@ionic-native/device": "4.14.0",
     "@ionic-native/file": "4.14.0",
+    "@ionic-native/in-app-browser": "^5.15.0",
```

Comment:
We should keep the same version for all @ionic-native plugins. That should be fixed to `4.14.0`